app/controllers/lookup_keys_controller.rb

```diff
@@ -34,4 +33,8 @@ def destroy
   def controller_permission
     'external_variables'
   end
+
+  def resource
```

Comment:
no, didn't find anything similar. this isn't relevant for anything other than this.
That said, I don't like using instance_variable_get, as it opens a new interpreter to eval the "@#{resource_name}") string...